lib/collective/collectors/postgres.rb

```diff
@@ -0,0 +1,42 @@
+module Collective::Collectors
+  class Postgres < Collective::Collector
+    MEGABYTE = 1024 * 1024
```

Comment:
I did it the way I did to avoid parsing and interpreting strings like "29 GB". Converting a non-prettified number of bytes into MB and adding MB as a unit at the end is also much more fine-grained and precise than the results returned by the query above, which round to the nearest GB.